sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -422,14 +425,18 @@ object CatalogTable {
  */
 case class CatalogStatistics(
     sizeInBytes: BigInt,
+    deserFactor: Option[Int] = None,
```

Comment:
Why not `Double`?

Reply:
It is expected to be a bigger number because of the columnar nature of the underlying file. Moreover it is better to take the ceil of a floating number here as being on the edge risks the an OOM.